setup.py

```diff
@@ -16,10 +17,10 @@
 import shutil
 import sys
 
-import pkg_resources
-from setuptools import Command, find_packages, setup
+import numpy
```

Comment:
I think the addition of this line has broken my package (internal to my organisation) - what happens if one tries to install pandas without numpy already installed?

Reply:
I'm using python 3.6, and it appears that there's no wheel for it on PyPI (only 3.7+) - that, in conjunction with this change means when I `pip install pandas` the source distribution is downloaded and its `setup.py` run to determine its dependencies (as opposed to having them statically defined, as would be in a wheel), and the whole thing falls over :/

Reply:
In this case, you should pin pandas version to <=1.1.5. pandas stopped py3.6 support starting from 1.2.0. pandas and many other numpy based scientific libraries are moving toward [NEP29](https://numpy.org/neps/nep-0029-deprecation_policy.html). So, if your work heavily depends on the numpy ecosystem, it might be a good time to think about moving to py3.7+.

Reply:
Yep, that seems the most sensible approach for me in the short term - thanks for the information, evidently I missed a memo 🙏